apps/web/app/(app)/environments/[environmentId]/integrations/slack/components/ManageIntegration.tsx

```
@@ -46,7 +46,12 @@ export const ManageIntegration = ({
   const handleDeleteIntegration = async () => {
     try {
       setisDeleting(true);
-      await deleteIntegrationAction({ integrationId: slackIntegration.id });
+      const deleteIntegrationActionResult = await deleteIntegrationAction({
+        integrationId: slackIntegration.id,
+      });
+      if (deleteIntegrationActionResult && deleteIntegrationActionResult.serverError) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)